bundles/com.espressif.idf.ui/plugin.xml

```diff
@@ -98,6 +98,11 @@
             id="com.espressif.idf.ui.command.eclipeguide"
             name="%command.name.9">
       </command>
+      <command
+            defaultHandler="com.espressif.idf.ui.handlers.UpdateEspIdfHandler"
+            id="com.espressif.idf.ui.command.update"
+            name="%command.name.14">
```

Comment:
Can you please name the property to reflect what it is referencing to meaningfully?

Reply:
Hi Ali, changed to the command.name.UpdateEspIdfMaster. Thanks